sycl/test/inline-asm/asm_16_empty.cpp

```diff
@@ -19,7 +19,8 @@ struct KernelFunctor : WithOutputBuffer<T> {
   void operator()(cl::sycl::handler &cgh) {
     auto C = this->getOutputBuffer().template get_access<cl::sycl::access::mode::write>(cgh);
     cgh.parallel_for<KernelFunctor<T>>(
-        cl::sycl::range<1>{this->getOutputBufferSize()}, [=](cl::sycl::id<1> wiID) [[cl::intel_reqd_sub_group_size(16)]] {
+        cl::sycl::range<1>{this->getOutputBufferSize()}, [=
+    ](cl::sycl::id<1> wiID) [[intel::reqd_sub_group_size(16)]] {
```

Comment:
clang-format does bad job on this constructs. I'd prefer it to be like this:
```suggestion
    // clang-format off
        cl::sycl::range<1>{this->getOutputBufferSize()}, 
    [=](cl::sycl::id<1> wiID) [[intel::reqd_sub_group_size(16)]] {
      // clang-format on
```

If you agree, please apply to all such cases.

Reply:
Yes, I agree. I will update all cases.